user_interface/lambda_function.py

```diff
@@ -250,42 +275,79 @@ def move_reminder_temp2active(user_reminder: UserReminder) -> None:
                 [user_reminder.temp_reminder])
                 )])
     bot.send_message(
-        chat_id, 
+        user_id, 
         text=f'Reminder {user_reminder.temp_reminder.name}'
             f' was added successfully')
+    user_reminder.temp_reminder.name = DEFAULT_NAME
+    user_reminder.update(actions=[
+                UserReminder.temp_reminder.set(user_reminder.temp_reminder)
+                ])
 
 
-def show_my_reminders(user_reminder: UserReminder) -> None:
+def show_reminders(user_reminder: UserReminder) -> None:
     """Returns reminder if it exists"""
-    chat_id = user_reminder.user_id
+    user_id = user_reminder.user_id
     reminders = []
-    for elem in user_reminder.active_reminders:
-        cur_data = f'Name={elem.name}. Amount per day={elem.amount}, amount to do today={elem.current_state}'
-        reminders.append(cur_data)
+    for reminder in user_reminder.active_reminders:
+        if not reminder.is_deleted:
+            cur_data = (f'Reminder "{reminder.name}", {reminder.amount} to do '
+                    f'per day. Today remains {reminder.current_state}.')
+            reminders.append(cur_data)
     if reminders:
-        message = "\n".join(reminders)
+        message = "\n\n".join(reminders)
         bot.send_message(
-            chat_id, 
+            user_id, 
             text=message)
     else:
-        bot.send_message(chat_id, text='Sorry, you have no reminders')
+        bot.send_message(user_id, text='Sorry, you have no reminders')
 
 
 def update_reminder(user_reminder: UserReminder, today_update: str) -> None:
+    """Updates reminder if it exists"""
     user_update = today_update.split()
-    if len(user_update) != 2 or not user_update[1].isdigit():
+    if (len(user_update) != 2 or not user_update[1].isdigit()):
```

Comment:
What does `2` indicate here?  This would be a good use of a constant as the variable name would help with readability.

Reply:
Fixed